django_celery_beat/models.py

```diff
@@ -186,12 +186,6 @@ class ClockedSchedule(models.Model):
         verbose_name=_('Clock Time'),
         help_text=_('Run the task at clocked time'),
     )
-    enabled = models.BooleanField(
```

Comment:
this is a breaking change

Reply:
> can you explain more of your reason for this approach?

I maybe handle it before this weekends(timezone: Beijing). too busy now.

Reply:
we can introduce swapalbe/ abstract base model as Django-activity-stream did

Reply:
Or we can keep it but it always True.

Reply:
what about default=False?

Reply:
But it act as always True.